tests/foreman/ui/test_provisioningtemplate.py

```diff
@@ -134,8 +132,6 @@ def test_positive_end_to_end(module_org, module_location, template_data, target_
         }
     ]
     with target_sat.ui_session() as session:
-        session.organization.select(org_name=module_org.name)
```

Comment:
@Gauravtalreja1 please update the tests from Puppet which has the same pattern

Reply:
yes, will be updating every test that runs UI tests with  `ui_session` to not select org/loc explicitly, along with updates to org/loc create fixtures too, when @jyejare @JacobCallahan agrees to this